packages/vue/src/errorhandler.ts

```diff
@@ -30,27 +30,15 @@ export const attachErrorHandler = (app: Vue, options: VueOptions): void => {
     setTimeout(() => {
       captureException(error, {
         captureContext: { contexts: { vue: metadata } },
-        mechanism: { handled: false },
+        mechanism: { handled: !!originalErrorHandler, type: 'vue-errorHandler' },
```

Comment:
Typically we just use the framework name as the mechanism type, so:

```suggestion
        mechanism: { handled: !!originalErrorHandler, type: 'vue' },
```

We have a [brief comment about this in relay](https://github.com/getsentry/relay/blob/99dcdd55c0e93dd30d0d5ecf5d354f3eb3935354/relay-event-schema/src/protocol/mechanism.rs#L104), but I'll make a PR to develop to make this more clear.

Reply:
That's my "bad" - I didn't know that this was more than coincidence in the places where we set a custom `type`. Thanks for pointing this out and develop sounds good!